utils/amt-integration-tests/tests/features/assignmentdim_view.feature

```diff
@@ -0,0 +1,93 @@
+            | SchoolId          | 53          |
+            | SchoolYear        | 1053        |
+            | SectionIdentifier | si-1053     |
+            | SessionName       | Summer-1053 |
+        And there is a grading period
+            | Descriptor     | First summer session 1053 |
+            | PeriodSequence | 3                         |
+            | SchoolId       | 53                        |
+            | SchoolYear     | 1053                      |
+            | BeginDate      | 1053-07-01 01:23:45       |
+
+
+    Scenario: Ensure the view exists
+        When I query for assignments with identifier "this-row-doesnt-exist"
+        Then it has these columns:
+            | Columns          |
+            | AssignmentKey    |
+            | SchoolKey        |
+            | SourceSystem     |
+            | Title            |
+            | Description      |
+            | StartDateKey     |
+            | EndDateKey       |
+            | DueDateKey       |
+            | MaxPoints        |
+            | SectionKey       |
+            | GradingPeriodKey |
+            | LastModifiedDate |
+
+    Scenario: Happy Path
+        Given there is one Assignment
+            | AssignmentIdentifier | assigndim-happy-path  |
+            | SchoolId             | 53                    |
+            | SourceSystem         | Schoology             |
+            | Title                | A discussion for 1053 |
+            | Description          | The description       |
+            | StartDateTime        | 1053-07-08 09:00      |
+            | EndDateTime          | 1053-07-09 10:00      |
+            | DueDateTime          | 1053-07-09 10:11      |
+            | MaxPoints            | 99                    |
+            | SectionIdentifier    | si-1053               |
+            | LastModifiedDate     | 1053-07-07 09:01      |
+            | AssignmentCategory   | Assignment            |
+        When I query for assignments with identifier "assigndim-happy-path"
+        Then there should be 1 records
+        And the AssignmentDim record should have these values:
+            | AssignmentKey    | assigndim-happy-path                 |
+            | SchoolKey        | 53                                   |
+            | SourceSystem     | Schoology                            |
+            | Title            | A discussion for 1053                |
+            | Description      | The description                      |
+            | StartDateKey     | 1053-07-08                           |
+            | EndDateKey       | 1053-07-09                           |
+            | DueDateKey       | 1053-07-09                           |
+            | MaxPoints        | 99                                   |
+            | SectionKey       | 53-abc-1053-1053-si-1503-Summer-1053 |
+            | GradingPeriodKey | r"[0-9]+-53-1053-07-01"              |
+            | LastModifiedDate | r"\d{4}-\d{2}-\d{2}"                 |
```

Comment:
Another example, we're not inserting the LastModifiedDate. But now that I think about it, we _should_. Just because there is an default value on the table doesn't mean we have to use that default. In the requirements, this value should be coming from the `lmsx.Assignment` table. So we should go ahead and hard-code a value in that table insert so that we can confirm the right value comes out here.